src/main/webapp/themes/fenixedu-learning-theme/dissertation.html

```diff
@@ -14,19 +14,19 @@
 {% endblock header %}
 
 {% macro orientatorCard(advisor, size, misteryMan) %}
-    {{ personCard(advisor.name, advisor.person.homepageWebAddress, advisor.person is null ? misteryMan : advisor.person.user.profile.avatarUrl , advisor.type.name, advisor.affiliation, advisor.category, size) }}
+    {{ personCard(advisor.name, advisor.person.homepageWebAddress, advisor.person is null ? misteryMan : advisor.person.user.profile.fullAvatarUrl , advisor.type.name, advisor.affiliation, advisor.category, size) }}
 {% endmacro %}
 
 {% macro studentCard(student, size) %}
-    {{ personCard(student.person.presentationName, student.person.homepageWebAddress, student.person.user.profile.avatarUrl, 'STUDENT', null, null, size) }}
+    {{ personCard(student.person.presentationName, student.person.homepageWebAddress, student.person.user.profile.fullAvatarUrl, 'STUDENT', null, null, size) }}
 {% endmacro %}
 
-{% macro personCard(name, homepageWebAddress, avatarUrl, type, affiliation, category, size) %}
+{% macro personCard(name, homepageWebAddress, fullAvatarUrl, type, affiliation, category, size) %}
```

Comment:
I believe it wasn't necessary to change this reference and the one below but i thought is was more consistent this way